app/_locales/en/messages.json

```diff
@@ -2986,6 +2986,9 @@
   "openSeaNew": {
     "message": "OpenSea"
   },
+  "operationCompletedSuccessfully": {
```

Comment:
Shall we be more explicit here and use something specific to the page like `resultPageSuccessDefaultMessage`?

Reply:
100% I pushed also one for the `resultPageErrorDefaultMessage`